esp/services/ws_store/ws_storeService.cpp

```diff
@@ -223,7 +223,7 @@ bool CwsstoreEx::onDeleteNamespace(IEspContext &context, IEspDeleteNamespaceRequ
 
     if (!global && !isEmptyString(targetUser))
     {
-        ESPLOG(LogMin, "CwsstoreEx::onDeleteNamespace: '%s' requesting to delete namespace on behalf of '%s'", user, targetUser);
+        AERRLOG("CwsstoreEx::onDeleteNamespace: '%s' requesting to delete namespace on behalf of '%s'", user, targetUser);
```

Comment:
This doesn't seem like an error message, and misclassifying non-errors as errors might not be a good idea.

Reply:
good point, this really should use LOG(MCauditWarning